decidim-budgets/app/helpers/decidim/budgets/projects_helper.rb

```diff
@@ -80,6 +83,45 @@ def current_rule_description
           t(".vote_threshold_percent_rule.description", minimum_budget: budget_to_currency(current_order.minimum_budget))
         end
       end
+
+      # Serialize a collection of geocoded projects to be used by the dynamic map component
+      #
+      # geocoded_projects - A collection of geocoded projects
+      def projects_data_for_map(geocoded_projects)
+        geocoded_projects.map do |project|
+          project_data_for_map(project)
+        end
+      end
+
+      def project_data_for_map(project)
+        project
+          .slice(:latitude, :longitude, :address)
+          .merge(
+            title: decidim_html_escape(translated_attribute(project.title)),
+            description: html_truncate(decidim_sanitize_editor(translated_attribute(project.description)), length: 100),
+            icon: icon("projects", width: 40, height: 70, remove_icon_class: true),
```

Comment:
I think the icon is not working correctly because this is what I see on the budgeting projects map:
![Budgeting projects map](https://user-images.githubusercontent.com/864340/170103431-0a1689d6-e821-4597-b11d-91ce14a74e1a.png)

Where as this is what I see on the proposals map:
![Proposals map](https://user-images.githubusercontent.com/864340/170103481-be5d2914-8134-482b-afd2-799d096b5e46.png)

I think the correct icon is `project` in singular format? Although it looks a bit weird icon to me.